pkg/scaffold/v1/webhook/webhook_test.go

```diff
@@ -39,39 +39,44 @@ var _ = Describe("Webhook", func() {
 	serverName := "default"
 	inputs := []*webhookTestcase{
 		{
-			Resource: resource.Resource{Group: "crew", Version: "v1", Kind: "FirstMate", Namespaced: true, CreateExampleReconcileBody: true},
+			Resource: resource.Resource{Group: "crew", Version: "v1", Kind: "FirstMate",
```

Comment:
Better to put all Resource fields on their own line for consistency.

Reply:
if we put all in one line it is long and will no pass in the lint.
It is harder understanding if we need to use the scroll.

Reply:
I'd do this which IMO read better. I guess that's also what @estroz means.
```
Resource: resource.Resource{
    Group: "crew",
    Version: "v1",
    Kind: "FirstMate",
    Namespaced: true,
    CreateExampleReconcileBody: true
},
```

Reply:
It makes sense. I did not understand that. Tks.

Reply:
Done